Exiled.API/Features/Core/Generic/EBehaviour.cs

```diff
@@ -60,6 +60,8 @@ protected EBehaviour()
         /// </remarks>
         protected virtual void FindOwner()
         {
+            // @Nao T is a GameEntity, why not add in inisde of it the method a abstract.
```

Comment:
If Get's are static, you shouldn't need to get it using `GetMethod`. Unless directly calling it through the class doesn't work, you shouldn't need to make reflection calls.

Reply:
Sorry, exiles use CSharplang 9 and the static interface method is only available in 11.

So if using reflection is the solution. Can we add a check like this.
```cs
            if (typeof(T).IsAssignableFrom(method.ReturnType))
```
And throw if false. This will help to better understand what is wrong and avoid null values in `Owner`.